sigs/contributor-strategy.md

```diff
@@ -0,0 +1,173 @@
+* Ken Owens
+* Cheryl Hung
+* Amye Scavarda Perrin
+* Ihor Dvoretskyi
+
+
+## Introduction
+This charter describes the operations of the CNCF Special Interest Group (SIG)
+Contributor Strategy. This SIG is responsible for contributor experience,
+sustainability, governance, and openness guidance to help CNCF community groups
+and projects with their own contributor strategies for a healthy project.
+
+Our initial three stakeholders:  
+1 - CNCF projects and their contributors/maintainers,  
+2 - End Users in the broader community and member companies,  
+3 - TOC
+
+## Mission
+Consistent with the CNCF SIG definition, the mission of CNCF SIG Contributor
+Strategy is to collaborate on strategies related to building, scaling, and
+retaining contributor communities, including (people) governance, communications,
+operations, and tools. To do that we will:
+* Create intentional space. Form a "Maintainers Circle" (name may
+  change) comprised of those interested in growing their projects and joining
+  fellow maintainers in related cross project discussions.
+* Listen and Advise. Create informational and training resources including
+guides, tutorials and templates of best practices, trade-offs, strategies,
+building and participating in scalable contributor communities.
+* Evaluate and Foster. Helping the TOC with assessments and due diligence of
+prospective new projects by developing community graduation criteria check
+points for rolling feedback and guidance.
+* Educate and Engage. Providing guidance to end users on how to engage with
+ contributors and vice versa.
+
+#### In scope:
+The following, non exhaustive, bootstrap list of activities and deliverables are
+in-scope for the SIG:
+* Definition of a contributor. This is helpful across projects for metrics and
+establishing guidelines, programs, and workflows.
+* “Contributing health checks”/’community health checks’ (name tbd) for project
+evaluations at graduation time.
+* Webinars, meetings, and other events to engage with the end user community on
+upstream contributing trainings and engagement programs.  
+* Development of guidelines, documents for project governance, recruiting and
+retaining contributor communities, mentorship, and project maturity.
+* Collection of current state of contributor strategies and governance models
+via surveys, GB reps, and Maintainers Circle (Example: what is the project doing
+  now, challenges, gaps)
+
+#### Out of scope
+* The day to day operations of CNCF SIGs, Kubernetes SIGs, or any community group
+of CNCF or its respective projects of any graduation level.
+* The creation and approval of CNCF SIGs or other community groups; we will offer advice but the responsibility lies on the TOC for those matters.
+* CNCF operations and marketing initiatives such as: product review/demo
+webinars, kubecon event planning, branding, stickers, swag, etc
+* Licensing and legal matters
+* Testing
+
+
+## Roadmap
+###1) SIG Formation:
+Role creation
```

Comment:
Any particular roles?  (e.g.  is this the TOC-specified roles of chair/tech lead, or more like [SIG-Security roles](https://github.com/cncf/sig-security/blob/master/governance/roles.md) which I would have described as "define how SIG members will step up to lead specific projects to accomplish our mission and facilitate the activities of the SIG" or something like that

Reply:
I went with the TOC-specified roles but after reviewing yours, I think we can use a ton of it. Lets see what others say.